From 23d84c13df47e47fc1cbccdedfae24b0974ff098 Mon Sep 17 00:00:00 2001
From: Dave Gerlach <d-gerlach@ti.com>
Date: Wed, 11 Dec 2013 01:47:14 +0000
Subject: [PATCH 0717/1115] ARM: OMAP2+: AM43XX: Allow setup reset of gpmc
 hwmod

Without initial soft reset of GPMC during hwmod setup the CLKCTRL
for GPMC will not transition to idle state which prevents PER_PD
transition so this patch allows it.

Signed-off-by: Dave Gerlach <d-gerlach@ti.com>
Acked-by: Russ Dill <russ.dill@ti.com>
(cherry picked from commit 8b33ff3367179c492859009f5ed092407614acea)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm/mach-omap2/omap_hwmod_33xx_data.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-omap2/omap_hwmod_33xx_data.c b/arch/arm/mach-omap2/omap_hwmod_33xx_data.c
index 8dc522b..550c24b 100644
--- a/arch/arm/mach-omap2/omap_hwmod_33xx_data.c
+++ b/arch/arm/mach-omap2/omap_hwmod_33xx_data.c
@@ -857,7 +857,6 @@ static struct omap_hwmod am33xx_gpmc_hwmod = {
 	.name		= "gpmc",
 	.class		= &am33xx_gpmc_hwmod_class,
 	.clkdm_name	= "l3s_clkdm",
-	.flags		= HWMOD_INIT_NO_RESET,
 	.prcm		= {
 		.omap4	= {
 			.modulemode	= MODULEMODE_SWCTRL,
-- 
1.7.5.4

